inst/tests/tests.Rraw

```diff
@@ -9266,13 +9266,17 @@ test(1658.8, fwrite(data.table(a=c(1:5), b=c(1:5)), quote=TRUE),
 test(1658.9, fwrite(data.table(a=c(1:3), b=c(1:3)), quote=TRUE),
             output='"a","b"\n1,1\n2,2\n3,3')
 
-# block size one bigger than number of rows
-test(1658.11, fwrite(data.table(a=c(1:3), b=c(1:3)), quote=TRUE),
-            output='"a","b"\n1,1\n2,2\n3,3')
+# fwrite output to console ignore compress
+test(1658.11, fwrite(data.table(a=c(1:3), b=c(1:3)), compress="gzip"),
```

Comment:
1. Why the old unit test was removed?
2. We could warn in such case

Reply:
If so, that should be mentioned somewhere, at least in PR comment. Otherwise it looks like PR can introduce breaking changes.
Giving new numbers might be still better way to go, as removed unit test numbers might be referenced somewhere else in the code or in github issues, so if you re-use those, it might result in confusion which will lead to `git blame` and potentially waste time tracking that. I would keep old tests and create new ones. As you are adding tests for new feature it is perfectly justified to not just add `1658.max+1` be append new tests at the end of tests.Rraw unit tests using new number.